publishing-spec/SPEC.md

```diff
@@ -1,194 +0,0 @@
-## dApp Store NFT Overview - v0.2.4
```

Comment:
This is referenced by the top-level README.md for the repo. Even if we want the schema to live elsewhere, IMO we should still have this file to (a) explain the schema, (b) link to the schema, and (c) provide a sample of what schema-compliant data looks like, since schemas can be tough to read and interpret if you aren't a SME in what is being validated.